doc/foreign-code.rst

```diff
@@ -53,47 +53,47 @@ the C/C++ source files or in the same directory group when using
 
 The header files must have the ``.h`` extension.
 
-Installing header files
+Installing Header Files
 -----------------------
 
 It is sometimes desirable to install header files with the
 library. For that you have two choices: install them explicitly with
 an :ref:`install` stanza or use the ``install_c_headers`` field of the
 :ref:`library` stanza. This field takes a list of header files names
-without the ``.h`` extension. When a library install header files,
-these are made visible to users of the library via the include search
+without the ``.h`` extension. When a library installs header files,
+they are made visible to users of the library via the include search
 path.
 
 .. _ctypes-stubgen:
 
 Stub Generation with Dune Ctypes
 ================================
 
-Beginning in dune 3.0, it is possible to use the ctypes_ stanza to generate
+Beginning in Dune 3.0, it's possible to use the ``ctypes_`` stanza to generate
```

Comment:
When you quote `ctypes_`, does the link to the ctypes library get destroyed?

Reply:
I think it does! Thank you for pointing that out.